nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -2331,23 +2336,23 @@ public void removeControllerService(final ControllerServiceNode service) {
             // and notify the Process Group that a component has been modified. This way, we know to re-calculate
             // whether or not the Process Group has local modifications.
             service.getReferences().getReferencingComponents().stream()
-                    .map(ComponentNode::getProcessGroupIdentifier)
```

Comment:
Likely IntelliJ auto-formatting again. You'll see a lot of this in this PR, likely, because classes were moved from one module to another, and I'm sure that IntelliJ applied some formatting when that was done.